src/eduid_common/api/decorators.py

```diff
@@ -29,45 +29,19 @@ def require_eppn_decorator(*args, **kwargs):
     return require_eppn_decorator
 
 
-def _get_user():
-    eppn = session.get('user_eppn', None)
-    if not eppn:
-        abort(401)
-    # Get user from central database
-    try:
-        return current_app.central_userdb.get_user_by_eppn(eppn, raise_on_missing=True)
-    except UserDoesNotExist as e:
-        current_app.logger.error('Could not find user central database.')
-        current_app.logger.error(e)
-        abort(401)
-    except MultipleUsersReturned as e:
-        current_app.logger.error('Found multiple users in central database.')
-        current_app.logger.error(e)
-        abort(401)
-
-
 def require_user(f):
     @wraps(f)
     def require_user_decorator(*args, **kwargs):
-        user = _get_user()
+        user = get_user()
         kwargs['user'] = user
         return f(*args, **kwargs)
     return require_user_decorator
 
 
-def require_dashboard_user(f):
-    @wraps(f)
-    def require_dashboard_user_decorator(*args, **kwargs):
-        user = get_dashboard_user()
-        kwargs['user'] = user
-        return f(*args, **kwargs)
-    return require_dashboard_user_decorator
-
-
 def require_support_personnel(f):
     @wraps(f)
     def require_support_decorator(*args, **kwargs):
-        user = _get_user()
+        user = get_user(no_private_userdb=True)
```

Comment:
Is this easier to parse as private_userdb=False?

Reply:
The get_user code actually saves the user in the private database with the correct modified_timestamp and that is what we don't want to do if no_private_userdb=True.

I don't feel strongly for or against no_private_userdb=True vs private_userdb=False, so I'll change it.